### DISCUSSION TITLE
Setting up emails + users able to register [paid work]

### DISCUSSION THREAD

**Discussion by JonLittleIT** [JordanKnott/taskcafe#176] [Ideas]
Anyone interested in making updates to allow for emails and user registration?

- Create a code change to allow users to sign up users with emails and create a login that new users can login with.
- Configure email service so when you sign up on registration task cafe emails new sign ups

I would be happy to pay for these changes if anyone has them and they can push there code.

**Comment by thomastthai** [comment]
@JonLittleIT, would you provide more details about those two items?
